repair/repair.cc

```diff
@@ -561,15 +561,15 @@ shard_repair_task_impl::shard_repair_task_impl(tasks::task_manager::module_ptr m
 
 void shard_repair_task_impl::check_failed_ranges() {
     rlogger.info("repair[{}]: shard {} stats: repair_reason={}, keyspace={}, tables={}, ranges_nr={}, {}",
-        id.uuid(), id.shard(), _reason, _status.keyspace, table_names(), ranges.size(), _stats.get_stats());
+        id.uuid(), _status.shard, _reason, _status.keyspace, table_names(), ranges.size(), _stats.get_stats());
```

Comment:
Is this the same as `this_shard_id()`? If so, we don't need to print this, the logger already includes the shard in each log message.